unit2.js

```diff
@@ -0,0 +1,68 @@
+document.addEventListener("DOMContentLoaded", () => {
+    let select = document.querySelector("select")
+    let form = document.querySelector("form")
+    let ul = document.querySelector("ul")
+
+    const getMovies = async () => {
+        try {
+            let res = await axios.get("https://ghibliapi.herokuapp.com/films")
+            let films = res.data;
+            // debugger
+            films.forEach(film => {
+                let option = document.createElement("option")
+                let selected = document.querySelector("select")
+                option.innerText = film.title;
+                option.value = film.url
+                // option.value = film.description;
+                select.appendChild(option)
+                selected.appendChild(option)
+                // debugger
+            })
+
+        } catch (err) {
+            console.log("error")
```

Comment:
This would just console log the string `error` instead of the actual error. It would more helpful to just console log the argument `err`